utils/forest_utils/src/cid/mod.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::{
+    multihash::{Code::Blake2b256, MultihashDigest},
+    Cid,
+};
+
+/// Extension methods for constructing `dag-cbor` [Cid]
+pub trait CidCborExt {
+    /// Default CID builder for Filecoin
+    ///
+    /// - The default codec is CBOR
+    /// - The default hash function is 256 bit BLAKE2b
+    ///
+    /// This matches [`abi.CidBuilder`](https://github.com/filecoin-project/go-state-types/blob/master/abi/cid.go#L49) in go
+    fn new_dag_cbor_default<S: serde::ser::Serialize>(obj: &S) -> anyhow::Result<Cid> {
```

Comment:
The naming feels non-intuitive to me. I can see that the `default` part refers to `blake256,` but I don't see any non-default counterpart. How about naming this function `from_cbor` or `from_cbor_blake256`?